ls1intum/Artemis
# `General`: Align the top position of both columns in the settings menu

### Checklist
#### General



#### Client

### Motivation and Context


This PR is related to and closes the issue [6183](https://github.com/ls1intum/Artemis/issues/6183).

### Description

The tops of the left and right columns on the settings page are not aligned in the same position, and there are some unnecessary horizontal lines on the Notifications tab under this page too. This PR aligns them correctly and removes the lines that are not needed.

### Steps for Testing

Prerequisites:
- 1 User (can be any type)

1. Log in to Artemis
2. Navigate to Settings
3. Confirm that the column alignment is correct
4. Open the Notifications tab
5. Confirm that unnecessary horizontal lines are deleted
6. Test the changes on both the dark and light scene

### Review Progress






#### Code Review
#### Manual Tests


### Screenshots



#### Before
<img width="626" alt="image" src="https://user-images.githubusercontent.com/81355607/233806946-4d79bc5f-e2ad-47dc-aa61-266a2413c95e.png">

#### After
<img width="624" alt="image" src="https://user-images.githubusercontent.com/81355607/233807001-d8d34c7e-db0a-4a00-abf5-5cd6cd04e767.png">
